README.md

```diff
@@ -40,10 +40,21 @@ Your first account will automatically get mod permission on both subreddits and
    In your application folder change all occurences of `pokemontrades` and `SVExchange` to the names of your testing subreddits. On Linux you can use:
    
    ```
-   find . -type f -exec sed -i 's/pokemontrades/mySubreddit/gI' {} \;
-   find . -type f -exec sed -i 's/SVExchange/mySubreddit1/gI' {} \;
+   find . -type f ! -path "./node_modules/*" ! -path "./.git/*" -exec sed -i "s/pokemontrades/firstsubname/g" {} \;
+   find . -type f ! -path "./node_modules/*" ! -path "./.git/*" -exec sed -i "s/PokemonTrades/FIRSTSUBNAME/g" {} \;
+   find . -type f ! -path "./node_modules/*" ! -path "./.git/*" -exec sed -i "s/svexchange/secondsubname/g" {} \;
+   find . -type f ! -path "./node_modules/*" ! -path "./.git/*" -exec sed -i "s/SVExchange/SECONDSUBNAME/g" {} \;
    ```
 
+   Before commiting you can revert the changes using (if you use `username` and `username1` as sub names make sure to reverse the order of commands so `username` is not overwritten):
+   ```
+   find . -type f ! -path "./node_modules/*" ! -path "./.git/*" -exec sed -i "s/secondsubname/svexchange/g" {} \;
```

Comment:
Pull this out of the PR, feel free to put it in as another PR, but it's unrelated.